user_docs/en/userGuide.t2t

```diff
@@ -1220,6 +1220,19 @@ By default, NVDA will detect the formatting at the position of the System caret
 
 Enable this option while proof reading documents in applications such as Microsoft Word, where formatting is important.
 
+==== Report indents as ====
+When Report Line Indentation is enabled, the Report indents as combo box has three options.
+
+- Tones
+- Speech
+- Both Speech and Tones
+
+
+If speech is selected, when the level of indented text changes, NVDA will say something like "twelve spaces" or "four tabs."

```

Comment:
nit: That should be: "12 space" and "4 tab". Note the numerals and the singular nouns. Obviously, plural would be better, but having plural definitions for all symbols is overkill, so we don't.